source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
 
-## Checklist
+<div class='info'>
+Disclaimer: Home Assistant is not your security advisor.  Due to continually changing threat landscapes and individual configurations we recommend best practices.  The advice given here should be considered as a security baseline and should not represent your entire security profile.  
+</div>
 
-Here's the summary of what you *must* do to secure your Home Assistant system:
+### Initial Setup
+
+After installation we strongly recommend you perform the following tasks before using the system.  
+
+| Task | Justification |
+| --- | --- |
+| Setup [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication.  HTTP will allow eavesdropping on your passwords. |
+| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
+| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied. | Running older versions of Home Assistant can provide vulnerabilities which may be exploitable. |
```

Comment:
This only covers OS installation types.

Reply:
Looks like tables messed up the conversation.  Are you talking about:
- Secure Access Methods portion?  There is nothing to secure on HAOS.  The installation media is secure on its own, right?
- Ensure all updates are applied?   It is conceivable that an update is published between installation and first use.
Of course Setup Encrypted Access (now changed) is applicable on all platforms. 

Please elaborate.